zio-http/src/main/scala/zio/http/endpoint/Endpoint.scala

```diff
@@ -258,30 +303,53 @@ final case class Endpoint[Input, Err, Output, Middleware <: EndpointMiddleware](
   def outStream[Output2: Schema](implicit
     alt: Alternator[Output, ZStream[Any, Nothing, Output2]],
   ): Endpoint[Input, Err, alt.Out, Middleware] =
-    outStream[Output2](Status.Ok)
+    Endpoint(
+      input,
+      output = (self.output | ContentCodec.contentStream[Output2]) ++ StatusCodec.status(Status.Ok),
+      error,
+      doc,
+      mw,
+    )
+
+  /**
+   * Returns a new endpoint derived from this one, whose output type is a stream
+   * of the specified type for the ok status code.
+   */
+  def outStream[Output2: Schema](name: String)(implicit
```

Comment:
I think we need `inCodec` and `outCodec` variations that take a name.

Reply:
I think we don't because the name is stored in the codec. So you'd write `outStream(HttpCodec.content[X]("name"))` etc

Reply:
Ah right